File: python/implementation/4-2.py
n = int(input())
hour = 0
min = 0
sec = 0
count = 0
while(hour <= n):
    if sec == 59:
        if min == 59:
            hour += 1
            min = 0
            sec = 0
        else:
            min += 1
            sec = 0
    else:
        sec += 1
    if (str(hour) + str(min) + str(sec)).find('3') != -1:
        count += 1

print(count)

# 완전 탐색 문제

File: javascript/implementation/4-2.js
let input = require("fs").readFileSync("../test.txt").toString().split("\n");
let n = parseInt(input[0]);

let count = 0;

for (let h = 0; h < n + 1; h++) {
  for (let m = 0; m < 60; m++) {
    for (let s = 0; s < 60; s++) {
      if ((h.toString() + m.toString() + s.toString()).includes("3")) {
        count += 1;
      }
    }
  }
}

console.log(count);

// 숫자 -> 문자열 변환 : 숫자.toString()
// 문자가 포함되었는지 확인 : 문자열.includes(문자)
